management/product/2026-02-15 - Public Timeline Improvements.md

```
@@ -0,0 +1,192 @@
+- **Público**: Full width sin sidebar, grid de 2 paneles
+
+Esto causa que los hero buckets se vean más altos y estrechos en público (menos ancho horizontal disponible para la columna del timeline).
+
+### Solución
+Asegurar que el **ancho de la columna de timeline sea consistente** entre ambas vistas. El público tiene más espacio horizontal disponible (no tiene sidebar), así que el contenido debería centrarse con un `max-width` que replique el ancho que tiene en la vista privada.
+
+```
+Privado:                          Público:
+┌────┬────────────────────┐      ┌──────────────────────────┐
+│Side│  Timeline | Browser│      │   Timeline | Browser     │
+│bar │  (ancho X)         │      │   (mismo ancho X,        │
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)